packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -18,56 +18,53 @@
               <a
-                :key="Boolean(hasLinkToProjects).toString()"
+                :key="Boolean(currentProject).toString()"
                 class="font-medium"
-                :class="hasLinkToProjects ? 'text-indigo-500 hocus-link-default' :
+                :class="currentProject ? 'text-indigo-500 hocus-link-default' :
                   'text-gray-700'"
-                :role="hasLinkToProjects ? undefined : 'link'"
-                :href="hasLinkToProjects ? 'global-mode' : undefined"
-                :ariaDisabled="!hasLinkToProjects"
+                :role="currentProject ? undefined : 'link'"
+                :href="currentProject ? 'select-project' : undefined"
+                :ariaDisabled="!currentProject"
                 @click.prevent="clearCurrentProject"
               >
                 {{ t('topNav.global.projects') }}
               </a>
             </li>
             <li
-              v-if="props.gql?.currentProject"
+              v-if="props.gql.isGlobalMode && currentProject?.title"
               class="mx-2px align-middle inline-block"
               aria-hidden
             >
-              <i-cy-chevron-right_x16
-
-                class="icon-dark-gray-200"
-              />
+              <i-cy-chevron-right_x16 class="icon-dark-gray-200" />
             </li>
             <li class="inline-block">
               <!-- context for use of aria role and disabled here: https://www.scottohara.me/blog/2021/05/28/disabled-links.html -->
               <!-- the `href` given here is a fake one provided for the sake of assistive technology. no actual routing is happening. -->
               <!-- the `key` is used to ensure the role/href attrs are added and removed appropriately from the element. -->
               <a
-                :key="Boolean(hasLinkToCurrentProject).toString()"
+                :key="canClearTestingType.toString()"
                 class="font-medium"
-                :role="hasLinkToCurrentProject ? undefined : 'link'"
-                :href="hasLinkToCurrentProject ? 'choose-testing-type' : undefined"
-                :class="hasLinkToCurrentProject ? 'text-indigo-500 hocus-link-default' :
+                :role="canClearTestingType ? undefined : 'link'"
+                :href="canClearTestingType ? 'choose-testing-type' : undefined"
+                :class="canClearTestingType ? 'text-indigo-500 hocus-link-default' :
                   'text-gray-700'"
-                :ariaDisabled="!hasLinkToCurrentProject"
+                :ariaDisabled="!canClearTestingType"
                 @click.prevent="clearTestingType"
               >
-                {{ props.gql?.currentProject?.title }}
+                {{ currentProject?.title }}
               </a>
               <template
-                v-if="props.gql?.currentProject?.branch"
+                v-if="currentProject?.branch"
```

Comment:
Outer check on the `<li>` would mean we don't need this `v-if`, too.

Reply:
Ahhh it turns out this `v-if` is needed indeed. Although we have a currentProject, it may not necessarily have a branch. I.e. the user is working locally and hasn't created a git branch yet